xrpl/utils/txn_parser/utils/balance_parser.py

```diff
@@ -158,25 +157,25 @@ def get_node_balance_changes(
 
 
 def group_by_account(
-    balance_changes: List[BalanceChange],
-) -> List[BalanceChanges]:
+    computed_balance: List[ComputedBalance],
```

Comment:
This should probably `computed_balances` (plural) since we're dealing with a list